src/core/Config.cpp

```diff
@@ -337,44 +337,55 @@ void Config::setDefaultActionKeys() {
 	}
 }
 
-void Config::setActionKey(ControlAction actionId, int index, InputKeyId key) {
+void Config::setActionKey(const ControlAction actionId, uint32_t index, InputKeyId newKey) {
```

Comment:
It makes the value range check easier as >= can be used to check the whole meaningful range.